index.html

```diff
@@ -42,5 +45,6 @@ <h1 id="Display Name"></h1>
   <script src="./js/view.js"></script>
   <footer></footer>
 </body>
+<!-- todo: looks like someone forgot to remove this closing tag -->
 </section>
-</html>
\ No newline at end of file
+</html>
```

Comment:
Sorry about these weird changes; just random styling the editor automatically does.